frontends/verific/verificsva.cc

```diff
@@ -1073,6 +1073,12 @@ struct VerificSvaImporter
 				break;
 			}
 
+			if (inst->Type() == PRIM_SVA_MATCH_ITEM_TRIGGER || inst->Type() == PRIM_SVA_MATCH_ITEM_ASSIGN)
+			{
+				parser_error("SVA sequences with local variable assignments are currently not supported.\n");
```

Comment:
I think this should be part of `void parser_error(Instance *inst)` above, and we should still print instance type, name, and code location.

Reply:
updated now is it is in proper place, but displays in this case two-line error message, containing all info.